packages/components/src/ButtonGroup/ButtonGroup.module.css

```diff
@@ -0,0 +1,23 @@
+.buttonGroup {
+  @apply inline-grid;
+
+  grid-auto-flow: column;
+}
+
+.spacing1x {
+  column-gap: 0.5rem;
+  row-gap: 0.5rem;
```

Comment:
Did tailwind classes not work for these styles?

Reply:
Ah, yes, they work, and we can also collapse it into one style instead of setting `column` and `row` separately. Updated, thanks!

Reply:
(nit): Also for `grid-auto-flow`?